packages/jaeger-ui/src/components/TracePage/TraceStatistics/index.tsx

```diff
@@ -479,12 +376,26 @@ export default class TraceStatistics extends Component<Props, State> {
         {this.state.showPopup ? (
           <PopupSQL closePopup={this.togglePopup} popupContent={this.state.popupContent} />
         ) : null}
-        <table className="test1893">
-          <tbody className="DetailTraceTableTbody--TraceStatistics">
-            {this.renderTableHead()}
-            {this.renderTableData()}
-          </tbody>
-        </table>
+        <Table
+          className="span-table span-view-table"
+          columns={columns}
+          dataSource={isNoDetail}
+          pagination={{
+            total: isNoDetail.length,
+            pageSizeOptions: ['10', '20', '50', '100'],
+            showSizeChanger: true,
+            showQuickJumper: true,
+          }}
+          rowClassName={record => {
+            if (record.type === 'undefined') {
```

Comment:
I can understand column.type, but what is record.type?

Reply:
yes I think instead of record.type , writing row.type makes more sense

Reply:
row.type is not more clear - what are examples of different row types?

Reply:
then what would you suggest ?? Here each row of the table refers to a span right?

Reply:
Here each row means it is like a span ,